configure.ac

```diff
@@ -5476,6 +5476,20 @@ then
   [Define if you have struct stat.st_mtimensec])
 fi
 
+# checking for glibc extensions to langinfo.h
+#  _NL_TIME_FIRST_WEEKDAY isn't #define'd, so test it directly
+AC_MSG_CHECKING(for _NL_TIME_FIRST_WEEKDAY)
+AC_CACHE_VAL(ac_cv_have__nl_time_first_weekday,
+AC_COMPILE_IFELSE([AC_LANG_PROGRAM([[#include <langinfo.h>]], [[
+int main(int argc, char *argv[]) { return !!nl_langinfo(_NL_TIME_FIRST_WEEKDAY); }]])],
+  [ac_cv_have__nl_time_first_weekday=yes],[ac_cv_have__nl_time_first_weekday=no]))
+AC_MSG_RESULT($ac_cv_have__nl_time_first_weekday)
+if test "$ac_cv_have__nl_time_first_weekday" = yes
```

Comment:
``AS_VAR_IF()``

Reply:
I do not understand.

Reply:
We recently replaced most ``if test`` with proper macros.

```
AS_VAR_IF([ac_cv_have__nl_time_first_weekday], [yes], [
  AC_DEFINE([HAVE__NL_TIME_FIRST_WEEKDAY], [1],
    [Define to 1 if you have _NL_TIME_FIRST_WEEKDAY in langinfo.h])
])
```